cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/export/StateVariablesExportTest.java

```diff
@@ -638,9 +640,14 @@ private boolean test(ReadOnlyDataSource dataSource, boolean exportTp, boolean ex
                 .forEach(t -> t.setP(0.0).setQ(0.0));
 
         // Export original and with new SV
-        // comparison without extensions, only Networks
+        // comparison without extensions, only Networks, to avoid differences in CgmesControlAreas
         ExportOptions exportOptions = new ExportOptions().setSorted(true);
         exportOptions.setExtensions(Collections.emptySet());
+        // Also, if a default area was created during export to CGMES, remove it from the expected network
+        // The actual network will not have it, it is the original network directly exported as IIDM
+        if (!originalNetworkHasAreas) {
+            expected.getAreaStream().map(Area::getId).toList().forEach(a -> expected.getArea(a).remove());
+        }
```

Comment:
That part could be removed when the issue of the CGMES export altering the IIDM network will be tackled.